ProjectFinale/src/com/company/clase/Actiuni.java

```diff
@@ -0,0 +1,116 @@
+package com.company.clase;
```

Comment:
[Fix] clase sunt toate in java, aici poate ai vrut sa zici entitati/modele/pojouri